test.php

```diff
@@ -0,0 +1,25 @@
+<?php
```

Comment:
This file was just for fast local testing and could be remove, right?

Reply:
Yea, my bad, that one slipped on. Removed.